test/unit/specs/store/delegates.spec.js

```diff
@@ -1,103 +1,185 @@
-import setup from "../../helpers/vuex-setup"
-
-let instance = setup()
+import delegatesModule from "renderer/vuex/modules/delegates.js"
+import nodeMock from "../../helpers/node_mock.js"
+import BN from "bignumber.js"
 
 describe(`Module: Delegates`, () => {
-  let store
+  let module
+  let node
 
   beforeEach(() => {
-    let test = instance.shallow()
-    store = test.store
+    node = Object.assign({}, nodeMock)
+    module = delegatesModule({ node })
   })
 
   it(`adds delegate to state`, () => {
-    store.commit(`addDelegate`, {
-      owner: `foo`,
-      tokens: `10`
-    })
-    expect(store.state.delegates.delegates[0]).toEqual({
-      id: `foo`,
-      owner: `foo`,
-      tokens: `10`,
-      voting_power: `10.00`
-    })
-    expect(store.state.delegates.delegates.length).toBe(1)
+    let { mutations, state } = module
```

Comment:
It's good to be in the habit of using `const` always instead of `let` because it lowers the complexity of the code.  If `mutations` is declared `const` then the reader's mind knows that it's always the same no matter where it is in the function.  If it's declared `let` then it's not possible to know which value it's holding unless one also pays close attention to time and the order of assignments.  It creates more work for everyone who reads your code later and makes bugs more likely.

Reply:
We decided to go with const.

Reply:
Who decided that?